webview_panels/src/lib/altimate/altimate-components.d.ts

```
@@ -1,150 +1,596 @@
+export declare interface ViewsCodeModalArgs {
+    table: string;
+    column: string;
+    viewsType: ViewsTypes;
+    viewsCode: Record<string, [string, string][]>;
+    nodeType: string;
 }
 
-export {};
+declare type ViewsTypes = keyof typeof VIEWS_TYPE_COLOR;
+
+export { }
```

Biome

[error] This empty export is useless because there's another export or import.

This import makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)